markdown-mode.el

```diff
@@ -30,7 +30,6 @@
 
 ;; See the README.md file for details.
 
-
```

Comment:
why did you remove the form feeds? it's actually quite common to have those in elisp modules. i never quite figured out *why*, that said, but i don't see how this relates to the rest of the (otherwise good) changes in this PR.

Reply:
Ignorance and maybe OCD ;-)  I thought it was noise.  Since then I've read up on them.  Apparently they're sometimes (rarely) used as a kind of section marker.  I've rebased and dropped that commit.